app/controllers/UserProfileController.scala

```diff
@@ -205,17 +206,43 @@ class UserProfileController @Inject() (implicit val env: Environment[User, Sessi
    * Creates a team and puts them in the organization table.
    */
   def createTeam() = Action(parse.json) { request =>
-  val orgName: String = (request.body \ "name").as[String]
-  val orgDescription: String = (request.body \ "description").as[String]
+    val orgName: String = (request.body \ "name").as[String]
+    val orgDescription: String = (request.body \ "description").as[String]
 
-  // Inserting into the database and capturing the generated orgId.
-  val orgId: Int = OrganizationTable.insert(orgName, orgDescription)
+    // Inserting into the database and capturing the generated orgId.
+    val orgId: Int = OrganizationTable.insert(orgName, orgDescription)
 
-  Ok(Json.obj(
-    "message" -> "Organization created successfully!",
-    "org_id" -> orgId 
-  ))
-}
+    Ok(Json.obj(
+      "message" -> "Organization created successfully!",
+      "org_id" -> orgId 
+    ))
+  }
+
+  /**
+  * Grabs a list of all the teams in the tables,
+  * regardless of open or closed status.
+  */
+  def getTeams() = UserAwareAction.async { implicit request =>
+    val teams: List[Organization] = OrganizationTable.getAllTeams()
+
+    // Convert the list of organizations to JSON
```

Comment:
Generally I'd advise against comments like this. With a line like `val teamJson = Json.toJson(teams)`, it feels pretty clear that this line converts to JSON, which feel like we're just cluttering the code more than anything by repeating stuff! Sometimes functions really are just clear enough to have just the method comment! Something as simple as this looks good to me:
```
  /**
  * Grabs a list of all the teams in the database, regardless of open or closed status.
  */
  def getTeams() = UserAwareAction.async { implicit request =>
    val teams: List[Organization] = OrganizationTable.getAllTeams()
    Future.successful(Ok(Json.toJson(teams)))
  }